src/KMonad/Args/Cmd.hs

```diff
@@ -79,3 +100,64 @@ levelP = option f
     f = maybeReader $ flip lookup [ ("debug", LevelDebug), ("warn", LevelWarn)
                                   , ("info",  LevelInfo),  ("error", LevelError) ]
 
+-- | Allow the execution of arbitrary shell-commands
+cmdAllowP :: Parser DefSetting
+cmdAllowP = SAllowCmd <$> switch
+  (  long "allow-cmd"
+  <> short 'c'
+  <> help "Whether to allow the execution of arbitrary shell-commands"
+  )
+
+-- | Re-emit unhandled events
+fallThrghP :: Parser DefSetting
+fallThrghP = SFallThrough <$> switch
+  (  long "fallthrough"
+  <> short 'f'
+  <> help "Whether to simply re-emit unhandled events"
+  )
+
+-- | TODO what does this do?
+initStrP :: Parser (Maybe DefSetting)
+initStrP = optional $ SInitStr <$> strOption
+  (  long "init"
+  <> short 'i'
+  <> metavar "STRING"
+  <> help "TODO"
+  )
+
+-- | Key to use for compose-key sequences
+cmpSeqP :: Parser (Maybe DefSetting)
+cmpSeqP = optional $ SCmpSeq <$> option
+  (tokenParser keywordButtons <|> megaReadM (choice noKeywordButtons))
+  (  long "cmp-seq"
+  <> short 'c'
```

Comment:
I just tested the develop branch and noticed that the short argument `-c` is defined twice (= `--allow-cmd` and `--cmp-seq`).

Same for `-i`. This is short for `--init` and `--input`.

Reply:
Hah, you are right of course.  That's what I get for only testing the long options; fixed in #164

Reply:
That was quick! :smile: